src/Api/Domain.php

```diff
@@ -299,4 +303,94 @@ public function verify(string $domain)
 
         return $this->hydrateResponse($response, VerifyResponse::class);
     }
+
+    /**
+     * Returns a domain tracking settings.
+     *
+     * @param string $domain name of the domain
+     *
+     * @return TrackingResponse|array|ResponseInterface
+     */
+    public function tracking(string $domain)
+    {
+        Assert::stringNotEmpty($domain);
+
+        $response = $this->httpGet(sprintf('/v3/domains/%s/tracking', $domain));
+
+        return $this->hydrateResponse($response, TrackingResponse::class);
+    }
+
+    /**
+     * Updates a domain click tracking settings.
+     *
+     * @param string $domain name of the domain
+     *
+     * @return UpdateClickTrackingResponse|array|ResponseInterface
+     *
+     * @throws \Exception
+     */
+    public function updateClickTracking(string $domain, string $active)
+    {
+        Assert::stringNotEmpty($domain);
+        Assert::stringNotEmpty($active);
+        Assert::oneOf($active, ['yes', 'no', 'htmlonly']);
+
+        $params = [
+            'active' => $active,
+        ];
+
+        $response = $this->httpPut(sprintf('/v3/domains/%s/tracking/click', $domain), $params);
+
+        return $this->hydrateResponse($response, UpdateClickTrackingResponse::class);
+    }
+
+    /**
+     * Updates a domain open tracking settings.
+     *
+     * @param string $domain name of the domain
```

Comment:
Missing annotation for the 2nd argument

```suggestion
     * @param string $domain The name of the domain
     * @param string $active The status for this tracking (one of: yes, no)
```